packages/controller/src/resources/monitoring/system/dashboards/opstrace-overview.json

```diff
@@ -972,18 +977,18 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum by (namespace,code) (rate(grafana_datasource_request_duration_seconds_sum{datasource=\"metrics\"}[5m]) / rate(grafana_datasource_request_duration_seconds_count{datasource=\"metrics\"}[5m]))",
+          "expr": "sum by (job, status_code) (rate(cortex_api_proxy_request_duration_seconds_count{method=\"GET\", route=~\"api_v1_query.*\"}[5m])) OR on() vector(0)",
```

Comment:
Note: using `OR on() vector(0)` to fill in zeros for instances when there are no queries happening.  This prevents a "no data" panel, which to me usually indicates an error.  Thoughts?